src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -1250,7 +1250,7 @@ def __init__(self, *args, feature_size: int = 256, mask_feature_size: int = 256,
             feature_size (`int`, *optional*, defaults to 256):
                 The feature size (channel dimension) of the FPN feature maps.
             mask_feature_size (`int`, *optional*, defaults to 256):
-                The features (channels) of the target masks size \\C_{\epsilon}\\ in the paper.
+                The features (channels) of the target masks size \C_{\epsilon}\ in the paper.
```

Comment:
```suggestion
                The features (channels) of the target masks size \\(C_{\epsilon}\\) in the paper.
```
No here a parenthesis was missing only for the synatx for LaTeX rendering.

Reply:
added as suggested